client/scripts/pinMetaData.js

```diff
@@ -0,0 +1,41 @@
+require("dotenv").config();
+const fs = require("fs").promises;
+const path = require("path");
+const config = require("config");
+var axios = require("axios");
+
+const pinMetaData = async (metaDataFilePath) => {
+    const ipfsFilePath = path.join(__dirname, config.get("ipfsFile.location"));
+    const ipfs = await fs.readFile(ipfsFilePath, "utf8");
+    var ipfsData = JSON.parse(ipfs);
+    ipfsData = ipfsData[ipfsData.length - 1];
+    console.log(ipfsData);
+
+    var metadata = await fs.readFile(metaDataFilePath, "utf8");
+    var metadataJson = JSON.parse(metadata);
+    // console.log(metadataJson);
+    var data = JSON.stringify({
+        ipfsPinHash: ipfsData.IpfsHash,
+        name: metadataJson.name,
```

Comment:
Can help me make all metadata files be named `metadata.json`? Cos I wanna just pull all metadata files via a [filter](https://www.npmjs.com/package/@pinata/sdk#pinList-anchor:~:text=Metadata%20filter%20object%20formatting)

Alternative is to add a {"type": "metadata"} key-value pair for me to filter on, but this one abit more mafan for u